services/services.rst

```diff
@@ -23,36 +23,18 @@ SuMIRe/PFS PBworks wiki
 
 "SuMIRe/PFS PBworks" is the only official wiki system for PFS. 
 
-======
-Login and ToC
-======
-
 After logging in at `PBworks <http://sumire.pbworks.com/>`_, 
 you will get a project index page named 
-"Subaru Measurement of Images and Redshifts (SuMIRe)". 
-This page contains links to contents in the wiki, such as 
-
-* PFS Project Office (General links, including link to telecon indexes)
-* Documents (Link list to documents)
-* Meeting, conference, etc.  (Link list to meetings)
-* Mailing lists (List of avail lists)
-* PFS working groups (Member list)
-
-======
-Page indexes
-======
-
-For some continuous meetings, index for each agenda/memo pages are avail.
-
-* `Systems Engineering group telecon <http://sumire.pbworks.com/Systems-engineering-telecons>`_
-* `Manager group telecon <http://sumire.pbworks.com/Management-telecons>`_
-
+`"Subaru Measurement of Images and Redshifts (SuMIRe)" <https://sumire.pbworks.com/w/Subaru%20Measurement%20of%20Images%20and%20Redshifts%20%28SuMIRe%29>`_. 
+This page contains links to contents in the wiki, also 
+`SideBar <https://sumire.pbworks.com/w/SideBar>`_ is shown at right side of 
+every wiki page as list of links with short names. 
 
```

Comment:
, also -> and also
at right side -> on the right side
links -> the links

Reply:
do we need "the" for the last one??

Reply:
Perhaps...